Common/MVKCommonEnvironment.h

```diff
@@ -75,6 +75,9 @@ extern "C" {
 /** Directive to identify public symbols. */
 #define MVK_PUBLIC_SYMBOL        __attribute__((visibility("default")))
 
+/** Directive to make a public alias of another symbol. */
+#define MVK_PUBLIC_ALIAS(a, t)   asm(".globl _" #a "; _" #a " = _" #t "\n")
```

Comment:
Hmmm...no way to alias C function name symbols without invoking `asm()`?

All function aliasing references I can find are for C++, which I assume is why you couldn't use that? And the `__attribute__((alias))` seems to be available...but it seem to require the declaration to include function params.

I'm amazed you uncovered this technique. Where did you find it?

Reply:
> Hmmm...no way to alias C function name symbols without invoking `asm()`?
> 
> All function aliasing references I can find are for C++, which I assume is why you couldn't use that? And the `__attribute__((alias))` seems to be available...but it seem to require the declaration to include function params.

The other thing is that `__attribute__((alias))` doesn't work on Darwin. Clang won't let you use it--even though Mach-O has `N_INDR`. Go figure.

`using` only works on functions in a different `namespace`, and it just makes that name available in the current namespace. Defining a variable with `auto` would've defined a *function pointer*, when what I wanted was to make a *symbol* alias.

> I'm amazed you uncovered this technique. Where did you find it?

I came up with it myself.

Reply:
@cdavis5e 

Building for iOS chokes on a number of identical build errors in the _Create Dynamic Library_ build phase:

    Undefined symbols for architecture arm64:
      "_vkBindBufferMemory2KHR", referenced from:
          MVKInstance::initProcAddrs() in libMoltenVK.a(libMoltenVK.a-arm64-master.o)
          ...

which presumably means those symbols don't appear in the built library. I'm wondering if the `asm()` technique doesn't work on iOS either.

Reply:
It's the semicolon. On x86, it's a statement separator, but for ARM it's the comment character. I'll make a PR to fix it.

Reply:
#1022